plugin.rb

```diff
@@ -224,17 +224,18 @@ def accepted_answer
         {
           post_number: info[0],
           username: info[1],
+          excerpt: info[2]
         }
       end
     end
 
     def accepted_answer_post_info
       # TODO: we may already have it in the stream ... so bypass query here
-
-      Post.where(id: accepted_answer_post_id, topic_id: object.topic.id)
-          .joins(:user)
-          .pluck('post_number, username')
-          .first
+      post = Post.where(id: accepted_answer_post_id, topic_id: object.topic.id).joins(:user).first
+      excerpt = post.excerpt
```

Comment:
My understanding is that I can't use 'pluck' because 'excerpt' isn't a database field, it's a method on the Post class (I think). 

As for why I've called excerpt twice, no idea, I'll fix that